osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -110,14 +119,11 @@ private void updateMods()
             RankedLabel.Text = null;

```

Comment:
Crazy idea: Have the label fixed with text/colour and just hide/show it instead of constantly changing it.

Reply:
How do you do that? Also, having fixed colour on the label won't show the fade animation.

Reply:
You don't need a colour transition as it'd always be blue.
`RankedLabel.FadeIn(200)` or `RankedLabel.FadeOut(200)`, 200 being the time it takes to complete.